base/shell/explorer/trayprop.cpp

```diff
@@ -279,8 +279,18 @@ PropSheetProc(HWND hwndDlg, UINT uMsg, LPARAM lParam)
     {
         case PSCB_INITIALIZED:
         {
+            RECT pos;
+            RECT rcWork;
             hIcon = LoadIconW(hExplorerInstance, MAKEINTRESOURCEW(IDI_STARTMENU));
             SendMessageW(hwndDlg, WM_SETICON, ICON_BIG, (LPARAM)hIcon);
+ 
+            GetWindowRect(hwndDlg,&pos);
+            SystemParametersInfo(SPI_GETWORKAREA, 0, &rcWork, 0);
+            if (pos.bottom > rcWork.bottom) 
+                pos.top -= (pos.bottom - rcWork.bottom + 2);
```

Comment:
Why +2? And what if this causes an underflow?

Reply:
Greetings to all, and to Timo.
Well, it seemed to me that it was better positioned, but I think you'll be right, better not add anything.
All the best to you.